### PR TITLE
upgrade to k3d v4.4.7

### PR DESCRIPTION
**Pull request by kuritka**
- upgrade k3d to v4.4.7
 - upgrade to v1.21.2-k3s1 - if you explicitly set the k3s image, upgrade on your side.
 - necessary documentation changes

Signed-off-by: kuritka <kuritka@gmail.com>

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -130,7 +130,7 @@ deploy(){
     curl --silent --fail ${K3D_URL} | TAG=${K3D_VERSION} bash
 
     echo -e "\existing_network${YELLOW}Deploy cluster ${CYAN}$name ${NC}"
-    eval "k3d cluster create $name --wait $arguments --network $network $registryArg"
+    eval "k3d cluster create $name --wait $arguments --network $network $registryArg" --verbose
```
Comment (k0da) [inline run.sh]: Do we need to keep verbose?

Comment (somaritane) [reply run.sh]: I'd probably make it a configurable parameter.

**Review by k0da — APPROVED** [review]
LGTM

File: run.sh (hunk 1 above)
Comment (kuritka) [reply run.sh]: @somaritane , good idea 👍   we can already run `--verbose` as an argument.

see usage in temporary k8gb workflow : https://github.com/k8gb-io/k8gb/blob/upgrade_to_k3d-action_v1.5.0/.github/workflows/terratest.yaml#L27

**Pull request merged**